### PR TITLE
Remove mention of rights

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: daybed/schemas/validators.py [hunk 1]
```diff
@@ -139,8 +139,7 @@ def model_validator(request):
 def permissions_validator(request):
     """Verify that the permissions defined in the request body are valid:
          - tokens exists
-         - rights are valids
-
+         - permission names are valids
```
Comment (almet) [inline daybed/schemas/validators.py]: typo: valid.

File: daybed/permissions.py [hunk 2]
```diff
@@ -114,24 +114,22 @@ def _handle_pyramid_constants(self, creators):
...
+
+        if principals.intersection(self.model_creators):
+            current_permissions.add("create_model")
 
-        token_permissions = set()
+        if principals.intersection(self.token_creators):
+            current_permissions.add("create_token")
 
-        self.add_conf_rights(self.model_creators, principals,
-                             token_permissions, "create_model")
```
Comment (Natim) [inline daybed/permissions.py:130]: Why did you remove this?

Comment (Natim) [inline daybed/permissions.py:130]: I read wrong, it is there: https://github.com/spiral-project/daybed/pull/200/files#diff-da7b902eb9dad2706632fb163a7f1d46R125

**Comment by Natim** [comment]
Thank you for this.
